teps/0120-canceling-concurrent-pipelineruns.md

```diff
@@ -102,7 +102,326 @@ queueing, or more advanced concurrency strategies.
 
 ## Proposal
 
-TODO
+Add a new field `concurrency` to `trigger.spec`, for example:
+
+```yaml
+apiVersion: triggers.tekton.dev/v1beta1
+kind: Trigger
+spec:
+  bindings:
+  - name: reponame
+    value: $(body.repository.full-name)
+  template:
+    ref: ci-pipeline-template
+  concurrency:
+    params:
+    - name: reponame
+    key: $(params.reponame)
```

Comment:
I am not sure if I understand those two. What is the role of both `params: { name: reponame }` and `key: …` ?

Reply:
I agree this is redundant-- updated proposal for how we specify params